src/lib/core/api/data-api.ts

```diff
@@ -128,13 +159,29 @@ export const BarplotResponse = type({
       }),
     }),
     data: array(
-      type({
-        label: array(string),
-        value: array(number),
-      })
+      intersection([
+        type({
+          //DKDK label can be number too?
```

Comment:
The [API Spec](https://veupathdb.github.io/EdaDataService/api.html#resources:/apps/pass/visualizations/barplot:post) says label is an array of string.

What discussions have you had with @d-callan about this?  I see that the labels of variables such as `GEMS->Participant->Enrollment year` are being returned as numbers.

Should the API specs be changed or should the service return strings?

I'm just making sure everyone has had a chance to discuss this.

Reply:
@bobular, first of all, thank you for your review 👍 

As for this issue, I know that the API doc shows the Label as string/string[], however, from my limited tests, it always returned number/number[]. I did not specifically ask Danielle about this as there have been many discrepancies between the doc and the return. Thus, I used union type instead, but we should ask Danielle about this as well.